streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -980,6 +980,7 @@ private void checkIfUnexpectedUserSpecifiedConsumerConfig(final Map<String, Obje
         // add admin retries configs for creating topics
         final AdminClientConfig adminClientDefaultConfig = new AdminClientConfig(getClientPropsWithPrefix(ADMIN_CLIENT_PREFIX, AdminClientConfig.configNames()));
         consumerProps.put(adminClientPrefix(AdminClientConfig.RETRIES_CONFIG), adminClientDefaultConfig.getInt(AdminClientConfig.RETRIES_CONFIG));
+        consumerProps.put(adminClientPrefix(AdminClientConfig.RETRY_BACKOFF_MS_CONFIG), adminClientDefaultConfig.getLong(AdminClientConfig.RETRY_BACKOFF_MS_CONFIG));
```

Comment:
Can we update `StreamsConfigTest#consumerConfigMustUseAdminClientConfigForRetries` for this change, too? (And maybe update test name: `consumerConfigShouldContainAdminClientConfigsForRetriesAndRetryBackOffMsWithAdminPrefix`)

Reply:
done